Telegram/SourceFiles/platform/linux/notifications_manager_linux.cpp

```diff
@@ -108,13 +108,22 @@ NotificationData::NotificationData(
 	if (ranges::find(capabilities, qsl("actions")) != capabilitiesEnd) {
 		_actions << qsl("default") << QString();
 
-		// icon name according to https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html
-		_actions << qsl("mail-reply-sender")
-			<< tr::lng_notification_reply(tr::now);
-
 		connect(_notificationInterface.get(),
 			SIGNAL(ActionInvoked(uint, QString)),
 			this, SLOT(notificationClicked(uint)));
+
+		if (ranges::find(capabilities, qsl("inline-reply")) != capabilitiesEnd) {
```

Comment:
This was changed to `capabilities.contains` in other places in current `HEAD` commit , please change here too.

```suggestion
		if (capabilities.contains(qsl("inline-reply"))) {
```